src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,16 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        if (numbers.isEmpty()) {
+            throw new RuntimeException("Can't get min value from list: " + numbers);
+        }
+        return numbers.stream()
+                .map(e -> e.split(","))
```

Comment:
```suggestion
                .map(s -> s.split(","))
```
It's better to name this variables by first letter of Objest that you have in your Stream. Or it could be a short noun if it will be informative. Change it everywhere. Or one more option is the first letter of collection name(in this case numbers -> "i" or "n" or "number")